### ISSUE TITLE
StopIteration Error reading View-only Spreadsheet

### ISSUE THREAD

**Issue by jmnwong** [burnash/gspread#122]
Getting a StopIteration error when I'm shared a read-only (granted "Can view" or "Can Comment") Spreadsheet.  Can't open the spreadsheet for any worksheets.

```
Traceback (most recent call last):
  File "gdoc.py", line 20, in <module>
    url_list = get_urls(user, pw, doc)
  File "gdoc.py", line 6, in get_urls
    sh = gc.open(doc).sheet1
  File "/Users/Justin/anaconda/lib/python2.7/site-packages/gspread/models.py", line 157, in sheet1
    return self.get_worksheet(0)
  File "/Users/Justin/anaconda/lib/python2.7/site-packages/gspread/models.py", line 148, in get_worksheet
    self._fetch_sheets()
  File "/Users/Justin/anaconda/lib/python2.7/site-packages/gspread/models.py", line 66, in _fetch_sheets
    self._sheet_list.append(Worksheet(self, elem))
  File "/Users/Justin/anaconda/lib/python2.7/site-packages/gspread/models.py", line 170, in __init__
    'edit', element).get('href').split('/')[-1]
  File "/Users/Justin/anaconda/lib/python2.7/site-packages/gspread/models.py", line 211, in _get_link
    feed.findall(_ns('link')))
  File "/Users/Justin/anaconda/lib/python2.7/site-packages/gspread/utils.py", line 18, in finditem
    return next((item for item in seq if func(item)))
StopIteration
```

This works fine for "Can Edit".

**Comment by msuozzo** [comment]
Fixed in #137

**Comment by hadsed-genapsys** [comment]
@msuozzo I don't think that fixes it. Ideally there would be an error message that's helpful in figuring out the bug. This one took a while for us to track down, please re-open.
